src/control/server/init/setup_spdk.sh

```diff
@@ -17,9 +17,14 @@ else
 	PCI_WHITELIST="$_PCI_WHITELIST" NRHUGE="$_NRHUGE" \
 	TARGET_USER="$_TARGET_USER" "$scriptpath"
```

Comment:
Where does `$_TARGET_USER` come from?

Reply:
passsed in from the calling environment

Reply:
environment gets computed and passed in src/control/server/storage_nvme.go:prep()